exporter/opentelemetry-exporter-datadog/tests/test_datadog_exporter.py

```diff
@@ -144,6 +144,13 @@ def test_translate_to_datadog(self):
         # pylint: disable=invalid-name
         self.maxDiff = None
 
+        resource = Resource(
+            labels={
+                "key_resource": "some_resource",
+                "service.name": "resource_service_name",
```

Comment:
Would be good to add a test for when `span.attributes` also has values for the same keys. Given the ordering above, `span.attributes` should override the resource labels.

Reply:
makes sense, added